### DISCUSSION TITLE
New tab button won't create a ne wtabs

### DISCUSSION THREAD

**Discussion by Flashwalker** [aleksey-hoffman/sigma-file-manager#272] [Q&A]
What is the purpose oft he New tab button if it's actually doesn't creates new tabs?
![bjDzFFKLBa](https://github.com/aleksey-hoffman/sigma-file-manager/assets/2188215/4ec5ca3d-3105-4aab-bb6a-dd5651091a2e)

**Comment by gerroon** [comment]
I have that issue too, it seems very buggy with opening tabs on 1.7
